Sources/SwiftProtobuf/StringUtils.swift

```diff
@@ -50,21 +62,27 @@ internal func utf8ToString(bytes: UnsafePointer<UInt8>, count: Int) -> String? {
     if count == 0 {
         return String()
     }
-#if os(Linux)
-    // As of March, 2017, the NSString(bytes:length:encoding:)
-    // initializer incorrectly stops at the first zero character for
-    // Linux versions of Swift:
+#if !os(Linux) || swift(>=3.2)
```

Comment:
All I know is what's been publicly talked about on the mailing lists—I've heard core team members talk about 4.0 but I've never heard them reference a possible 3.2. If they were planning to do another 3.x point-release with bug fixes, I'd expect that they'd at least have a branch for it on Github though.